crates/ruma-common/Cargo.toml

```diff
@@ -13,7 +13,6 @@ edition = "2018"
 [dependencies]
 indexmap = { version = "1.6.2", features = ["serde-1"] }
 js_int = { version = "0.2.0", features = ["serde"] }
-maplit = "1.0.2"
```

Comment:
Was it a clippy lint that noticed this only being used as a dev-dependency? Either way, very nice! :)

Reply:
Yeah, this was `unused_crate_dependencies` which was cool but doesn't handle features well, it thought the entire ruma crate was empty so I couldn't keep it :disappointed: .

Reply:
We could also look into adding [cargo-udeps](https://github.com/est31/cargo-udeps) to CI (but in a separate PR).